packages/knip/src/plugins/dependency-cruiser/index.ts

```diff
@@ -0,0 +1,27 @@
+import type { IsPluginEnabled, Plugin } from '../../types/config.js';
+import { hasDependency } from '../../util/plugin.js';
+
+// https://github.com/sverweij/dependency-cruiser
+
+const title = 'dependency-cruiser';
+
+const enablers = ['dependency-cruiser'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+
+const config = ['.dependency-cruiser.{js,cjs,mjs,json}'];
+
+const args = {
+  binaries: ['depcruise'],
+  alias: { config: ['c'] },
+  string: ['config'],
+  config: true,
```

Comment:
setting `config: true` is a shorthand that sets the `alias` and `string` options as well: https://github.com/webpro-nl/knip/blob/5a77dcc47c2cdfc026a5296f1dc3cf54c786c562/packages/knip/src/types/args.ts#L59-L69

just a heads-up, no need to change if you prefer explicit

Reply:
Nice, I didn't know that